src/desktop-lite/README.md

```diff
@@ -29,6 +29,16 @@ This feature provides two ways of connecting to the desktop environment it adds.
 1. Open the ports view in your tool, select the noVNC port, and click the Globe icon.
 1. In the browser that appears, click the **Connect** button and enter the desktop password (`vscode` by default).
 
+To set up the `6080` port from your `devcontainer.json` file, include the following:
+```json
```

Comment:
Hi 👋 

Thanks for making this change. 

The README is auto-generated from `NOTES.md` and `devcontainer-feature.json` files by the devcontainers/action . See https://github.com/devcontainers/action

As this file will get overwritten, can you add the same contents to the https://github.com/devcontainers/features/blob/main/src/desktop-lite/NOTES.md instead? Thanks!